client/state/current-user/gravatar-status/actions.js

```diff
@@ -34,20 +41,40 @@ export function uploadGravatar( file, bearerToken, email ) {
 					dispatch( {
 						type: GRAVATAR_UPLOAD_REQUEST_SUCCESS
 					} );
+					dispatch( recordTracksEvent( 'calypso_edit_gravatar_upload_success' ) );
```

Comment:
Similarly it may make sense to attach this to `GRAVATAR_UPLOAD_REQUEST_SUCCESS`. Usage is a bit shorter here since there's only one stat to bump.
```jsx
dispatch( withAnalytics( 
    recordTracksEvent( 'calypso_edit_gravatar_upload_success' ),
   { type: GRAVATAR_UPLOAD_REQUEST_SUCCESS }
) );
```